cpp/ps0001/ps0001soln.cpp

```diff
@@ -1,14 +1,18 @@
 #include <iostream>
-#include <limits.h>
+
 using namespace std;
 
 int main()
 {
-  int a;
-  int b;
-  int c;
-  cin >> a >> b >> c ;
-  int s = a + b + c ;
-  cout << s ;
+  // prompting users for 3 numbers to add 1/2
+  short num1;
+  short num2;
+  short num3;
+  cin >> num1 >> num2 >> num3;
+
+  // outputting the result of the summation process 2/3
+  short sum = num1 + num2 + num3;
+  cout << sum;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```